aw-server/src/endpoints/mod.rs

```diff
@@ -88,24 +89,13 @@ fn get_device_id() -> String {
 
 #[get("/")]
 fn server_info() -> JsonValue {
-    let testing: bool;
-    #[cfg(debug_assertions)]
-    {
-        testing = true;
-    }
-    //TODO: Should this be commented?
-    #[cfg(not(debug_assertions))]
-    {
-        testing = false;
-    }
-
     let hostname = gethostname().into_string().unwrap_or("unknown".to_string());
     const VERSION: Option<&'static str> = option_env!("CARGO_PKG_VERSION");
 
     json!({
         "hostname": hostname,
-        "version": format!("aw-server-rust v{}", VERSION.unwrap_or("(unknown)")),
-        "testing": testing,
+        "version": format!("v{} (rust)", VERSION.unwrap_or("(unknown)")),
+        "testing": is_testing,
```

Comment:
This is not a good solution either as it won't work with an "--testing" option either.

The proper way would be to add it to the ServerState struct and make server_info read that.

Reply:
Sure, but as I mentioned in #102 I don't know the best way to do that. So can we merge this in the meanwhile?

Reply:
Sure!